### ISSUE TITLE
Handle npm scoped packages

### ISSUE THREAD

**Issue by prometheansacrifice** [esy/pesy#31]
Since pesy no longer operates in the build environment, scoped package names must be handled by the user (`@myscop/foo` => `myscope--foo`). This seem inconvenient and confusing. We must figure a way around this.

**Comment by alexcardell** [comment]
@prometheansacrifice Can you elaborate more on how to handle it as a user? I'm having trouble finding examples of using scoped packages with esy.

**Comment by prometheansacrifice** [comment]
In case your package.json looks like

```json
{
  "name": "@my-scope/pesy-npm",
  "version": "0.0.0",
  "description": "My Project"
  ...
}
```

you will have to mention the build command explicitly

```
{
  "name": "@my-scope/pesy-npm",
  "version": "0.0.0",
  "description": "My Project",
  "esy": {
    "build": "dune build -p my-scope--pesy-npm"
  }
}
```
The default `dune build -p #{self.name}` won't work.

Documentation: https://github.com/esy/pesy#workaround-for-scoped-packages-on-npm
Example (from the e2e test that run on CI): https://github.com/esy/pesy/blob/master/e2e-tests/pesy-configure-test-projects/pesy-npm-scoped/package.json

**Comment by ulrikstrid** [comment]
I think you can use `dune build --only-packages=package1,package2 root=./`. And we should know what the package names are as we set them in the dune files.

**Comment by prometheansacrifice** [comment]
Closed by #116
